### DISCUSSION TITLE
How to take memory snapshot of extension host in Remote-SSH?

### DISCUSSION THREAD

**Discussion by WesleyYue** [microsoft/vscode-discussions#1682] [Extension Development QnA] [answered]
- I am working on a VSCode extension
- There's a bug where memory usage grows quickly under certain conditions, but this only happens while running the extension in remote-ssh, where the extension is running on the SSH host.
- How do I do the equivalent of `code --inspect-extensions 9299` for the extension host that's installed on the remote server?

**Comment by isidorn** [comment]
Not sure. Maybe @connor4312 might be able to help

**Reply by connor4312** [reply, marked as answer]
You can clone your extension on your remote device, connect via remote - ssh, and debug it there. Then it'll run and be debugged in the remote extension host.
